lms/static/js/discovery/result_item_view.js

```diff
@@ -0,0 +1,50 @@
+;(function (define) {
+
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'gettext',
+    'date'
+], function ($, _, Backbone, gettext, Date) {
+    'use strict';
+
+    function formatDate(date) {
+        return dateUTC(date).toString('MMM dd, yyyy');
```

Comment:
@smagoun This is basically the same thing that @chrisndodge did for date formatting - seems like we are calling out for something more global within js - the more we pass around dates from json/ajax the more we'll need to consolidate.